### ISSUE TITLE
Table name of form exceeds 64 characters

### ISSUE THREAD

**Issue by robinbraibant** [verbb/formie#603]
**Description**
One of our users creates his own forms using Formie. When a new form is created, Formie uses the title of the form to create a handle. This handle is used to create a new table in the database.

When the user defines a long title that exceeds 64 characters (the max characters a table name can be in MySql), the handle and the table name will also exceed this amount.

When the form gets queried, the result is that the form cannot be found and the user gets an internal server error.

**Steps to reproduce**
1. Create a form with the title "22 & 29 november '21 - Suïcidepreventie voor volwassenenbegeleiders – Brussel"
2. Check the handle
3. Check the tablename
4. Try to query the form

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Page Reload
- Client-side Validation: Yes 
- Custom Form Templates: Yes 

**Additional info**
- Plugin version:1.4.20
- Craft version:3.7.16
- Multi-site: Yes

**Additional context**

**Comment by engram-design** [comment]
Looks like this wasn't applied to forms, only field handles. Fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.20",
  "...": "..."
}
```

Then run `composer update`.

**Comment by robinbraibant** [comment]
Thanks for the quick fix!

**Comment by engram-design** [comment]
Fixed in [1.4.21](https://github.com/verbb/formie/releases/tag/1.4.21)
